packages/backend/src/openapi/webhooks.yaml

```diff
@@ -160,7 +160,7 @@ components:
             - createdAt
             - updatedAt
             - expiresAt
-            - sendAmount
+            - debitAmount
```

Comment:
It seems like we have some validation error here. Check in "files changed" if you can't see it here.

Reply:
@sabineschaller Hmm, I see it. More details from the CI job show the warning:

    OpenAPI "servers" must be present and non-empty array
 https://github.com/interledger/rafiki/actions/runs/6174411421/job/16758949762?pr=1755#step:6:12

Looks unrelated to this change and we've been getting it for a while:
- 2 months ago: https://github.com/interledger/rafiki/commit/f5c1f6613de348ad0b266bb811c4b8cf5d729524#diff-90da7214f1ef2d21dca042e424b4079b684ab3017fb6c278e4abab4453855f24
- 3 months ago: https://github.com/interledger/rafiki/commit/22ef1dc9f398ed0a036c8fce9eed42f1d8825192#diff-90da7214f1ef2d21dca042e424b4079b684ab3017fb6c278e4abab4453855f24R1

Seems like it needs the `servers` attribute defined like we do in some of our other specs. Not sure what would be appropriate. What about something like this (similar to exchange rate yaml):

```yaml
servers:
  - url: 'https://account-servicing-entity.com/webhooks'
```

Reply:
Found the webhook url in some config files and concluded that url is right. I added it which seems to have fixed the warning.